Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>firebase/serviceAccountKey.json\n.env\n*storage\nnode_modules/
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision f50777a699e1fa5b4305630c64d5ccf743e36904)
+++ b/.gitignore	(date 1753136782707)
@@ -1,4 +1,11 @@
 firebase/serviceAccountKey.json
 .env
 *storage
-node_modules/
\ No newline at end of file
+node_modules/
+.DS_Store
+.idea/
+database/
+db/
+internal/
+models/
+vendor/
